src/core/DisplayGroup.h

```diff
@@ -40,16 +40,28 @@
 #define DISPLAY_GROUP_H
 
 #include "types.h"
-#include "Coordinates.h"
```

Comment:
I wouldn't remove this include because it is the parent class, so it's best to have it explicitly here. Even if it gets included through ContentWindow.